invenio_app_ils/ui/backoffice/src/common/api.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+// In development create a personal token to be able to perform requests and
+// and put it in a .env file under the name `REACT_APP_JWT_TOKEN`
+
+const getProductionToken = () => {
+  const res = document.getElementsByName('authorized_token');
+  if (res.length > 0 && res[0].hasOwnProperty('value')) {
+    return res[0].value;
+  }
+  return '';
+};
+
+const token =
+  process.env.NODE_ENV === 'production'
+    ? getProductionToken()
+    : process.env.REACT_APP_JWT_TOKEN;
+
+export const $axios = axios.create({
+  baseURL:
+    process.env.NODE_ENV === 'production'
+      ? '/api'
+      : 'https://127.0.0.1:5000/api',
```

Comment:
I think your tests are now calling the a real url because you have changed everything from `localhost` to `127.0.0.1` but in the tests the mocks are still:
`const loansBaseUrl = 'https://localhost:5000/api/circulation/loans';`

Anyway, keep in mind that this baseUrls will disappear when we integrate the flask-js lib: https://github.com/inveniosoftware/invenio-app-ils/issues/41